test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/Link.cshtml

```diff
@@ -12,6 +12,9 @@
     <!-- Globbed link tag with existing file -->
     <link asp-href-include="**/site.css" rel="stylesheet" title="&lt;the title>" />
 
+    <!-- Globbed link tag with comma separated pattern -->
+    <link asp-href-include="~/*.css, ~/sub/*.css" rel="stylesheet" title="&lt;the title" />
```

Comment:
(Not for fixing in this PR but) does `<link asp-href-include="~/joe.css, /joe.css, \\joe.css, joe.css" rel="stylesheet" title="&lt;the title" />` generate four identical `<link/>` elements?